### PR TITLE
Update upgrade tests to polarion compatible

### LINKED ISSUE #39: Update Upgrade Tests docstrings to get place in Polarion automatically
We need to update the docstrings of upgrade tests with following keywords to get them in polarion:

- [x] Update all uuid tests with 'upgrade-' prefix
- [x] Add ```subtype1: instalability``` in all test files
- [x] Update TestType to 'nonfunctional' from current 'NonFunctional'
- [x] Update CaseLevel to 'System' from 'Acceptance'.

### PR DESCRIPTION
**Pull request by jyejare**
Closes #39

### CODE REVIEW

File: upgrade_tests/test_existance_relations/test_architectures.py [hunk 1]
```diff
@@ -5,14 +5,16 @@
...
 
 :CaseComponent: CLI
 
-:TestType: NonFunctional
+:TestType: nonfunctional
 
 :CaseImportance: High
 
+:SubType1: installability
+
```
Comment (sghai) [inline upgrade_tests/test_existance_relations/test_architectures.py]: Curious what all other SubType1 we have ?

File: upgrade_tests/test_existance_relations/test_architectures.py [hunk 2]
```diff
@@ -28,7 +30,7 @@
 def test_positive_architectures_by_name(pre, post):
     """Test all architectures are existing after upgrade by names
 
-    :id: eb6d3728-6b0b-4cb7-888e-8d64a46e7beb
+    :id: upgrade-eb6d3728-6b0b-4cb7-888e-8d64a46e7beb
```
Comment (sghai) [inline upgrade_tests/test_existance_relations/test_architectures.py]: 👍

File: upgrade_tests/test_existance_relations/test_architectures.py (hunk 1 above)
Comment (jyejare) [inline upgrade_tests/test_existance_relations/test_architectures.py]: We have lot but not upgrade :)

Comment (jyejare) [inline upgrade_tests/test_existance_relations/test_architectures.py]: After discussion with @elyezer and @omaciel , We decided to keep it installability.

Comment (sghai) [inline upgrade_tests/test_existance_relations/test_architectures.py]: alright.. I was not aware of this tag.. may be I'll check w/ you offline

**Comment by sghai** [comment]
Looks good to me.. ACK

**Pull request merged**